src/vmm/src/vstate/memory.rs

```diff
@@ -800,119 +853,139 @@ mod tests {
     }
 
     #[test]
-    fn test_restore_memory() {
-        let page_size: usize = get_page_size().unwrap();
+    fn test_dump() {
```

Comment:
I would really love to see this test being decomposed to smaller tests.

Reply:
This is already spit up. `test_dump` and `test_dump_dirty` were one test before.